api/src/main/java/org/pmiops/workbench/auth/UserInfoService.java

```diff
@@ -27,9 +28,15 @@ public class UserInfoService {
   }
 
   public Userinfo getUserInfo(String token) {
-    GoogleCredential credential = new GoogleCredential().setAccessToken(token);
     Oauth2 oauth2 =
-        new Oauth2.Builder(httpTransport, jsonFactory, credential)
+        new Oauth2.Builder(
+                httpTransport,
+                jsonFactory,
+                new HttpRequestInitializer() {
```

Comment:
Do you mind elaborating on what is happening here? I don't understand what the initialize method is doing.

Reply:
I don't know how `GoogleCredential` was using the access token, but I do know that there isn't much you can do with one besides include it in the Authorization header. So, I just found a version of Oauth2.Builder that lets me set that header.